src/comitNode.ts

```diff
@@ -103,7 +103,7 @@ export class ComitNode {
       };
     }
 
-    log(
+    logger.debug(
```

Comment:
🙃Program flow logging.
```suggestion
    logger.trace(
```